src/main/java/service/writereadcsv/FruitServiceReaderCsvImp.java

```diff
@@ -0,0 +1,59 @@
+package service.writereadcsv;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import model.Fruit;
+
+public class FruitServiceReaderCsvImp implements FruitServiceReaderCsv {
```

Comment:
lets create separate classes FileReaderImpl that just reads data from file, not more